pkgs/utils/utils.go

```diff
@@ -208,3 +209,24 @@ func ValidateOpsLen(len int) error {
 		return fmt.Errorf("amount of operators should be 4,7,10,13: got %d", len)
 	}
 }
+
+func GetMessageHash(msg interface{}) ([32]byte, error) {
+	hash := [32]byte{}
+	switch msg := msg.(type) {
+	case *wire.ResignMessage:
+		msgBytes, err := msg.MarshalSSZ()
+		if err != nil {
+			return hash, err
+		}
+		copy(hash[:], eth_crypto.Keccak256(msgBytes))
+	case *wire.ReshareMessage:
+		msgBytes, err := msg.MarshalSSZ()
+		if err != nil {
+			return hash, err
+		}
+		copy(hash[:], eth_crypto.Keccak256(msgBytes))
+	default:
+		return hash, fmt.Errorf("unexpected message type: %T", msg)
+	}
+	return hash, nil
+}
```

Comment:
in the previous review I wrote you can use the `marshaller` interface and remove the `switch`

not a blocker, just nicer code

Reply:
applied in bulk PR #126 because there is another update for this function there